.github/workflows/release.yml

```diff
@@ -83,6 +83,14 @@ jobs:
         includes: '*.paa;*.xml'
         flags: '"-clear","-prefix=fr\gsri\vests"'
 
+    - name: Build GSRI Lockers
+      uses: team-gsri/actions-addon-builder@master
+      with:
+        source: 'addons/gsri_lockers'
+        target: 'addons'
+        includes: '*.paa;*.xml'

```

Comment:
sqf ?

Reply:
Shouldn't we standardize this kind of thing once and for all, with a proper inclusion list ? I'd rather have an optional exclusion list than a mandatory inclusion list to modify on each and every mod. I don't see the added value here ?

Reply:
sorry, this is how addon builder works :(
you can use `*.*` but you will have both config.cpp and config.bin in the pbo.